src/hooks/app/useSetUp.ts

```diff
@@ -69,10 +69,10 @@ export function useSetUp() {
         if (syncedAidropWithBackend && !isInitializingRef.current) {
             isInitializingRef.current = true;
             clearStorage();
-            invoke('setup_application').catch((e) => {
-                console.error(`Failed to setup application: ${e}`);
-                setCriticalError(`Failed to setup application: ${e}`);
-            });
+            // invoke('setup_application').catch((e) => {
```

Comment:
We have to wait with setting up until `set_airdrop_access_token` is called. It's important for setting up telemetry data

Reply:
can I emit an event `setup_complete` instead?

Reply:
We need to pass the access token from the frontend to the backend before the setup_application process.